src/App/DocumentObjectPyImp.cpp

```diff
@@ -880,6 +886,6 @@ Py::Boolean DocumentObjectPy::getNoTouch() const {
     return Py::Boolean(getDocumentObjectPtr()->testStatus(ObjectStatus::NoTouch));
 }
 
-void DocumentObjectPy::setNoTouch(Py::Boolean value) {
+void DocumentObjectPy::setNoTouch(const Py::Boolean& value) {

```

Comment:
The header DocumentObjectPy.h is a generated file and the value is passed by value.The `Py::` classes are a lightweight wrapper for PyObject similar to a smart pointer.

If this should be changed then the generation script must be changed and with it a lot of source files. If we decide to do it then this has to happen in a separate commit.

Reply:
OK! Will reverse. 'generated file' answers a couple of questions. Will be more cautious in future.